### ISSUE TITLE
Wrong file owner for files in deb package

### ISSUE THREAD

**Issue by sharkdp** [sharkdp/fd#213]
Currently, the linter for Debian packages shows that the files are not created by root (but by the Travis user with uid 2000):
```
> lintian fd_6.2.0_amd64.deb 
W: fd: missing-depends-line
E: fd: changelog-file-missing-in-native-package
E: fd: no-copyright-file
E: fd: extended-description-is-empty
E: fd: maintainer-address-missing David Peter
E: fd: wrong-file-owner-uid-or-gid usr/ 2000/2000
E: fd: wrong-file-owner-uid-or-gid usr/bin/ 2000/2000
E: fd: wrong-file-owner-uid-or-gid usr/bin/fd 2000/2000
E: fd: wrong-file-owner-uid-or-gid usr/share/ 2000/2000
E: fd: wrong-file-owner-uid-or-gid usr/share/bash-completion/ 2000/2000
E: fd: wrong-file-owner-uid-or-gid usr/share/bash-completion/completions/ 2000/2000
E: fd: wrong-file-owner-uid-or-gid usr/share/bash-completion/completions/fd 2000/2000
E: fd: wrong-file-owner-uid-or-gid usr/share/doc/ 2000/2000
E: fd: wrong-file-owner-uid-or-gid usr/share/doc/fd/ 2000/2000
E: fd: bad-owner-for-doc-file usr/share/doc/fd/ travis/travis != root/root
E: fd: wrong-file-owner-uid-or-gid usr/share/doc/fd/LICENSE-APACHE 2000/2000
E: fd: bad-owner-for-doc-file usr/share/doc/fd/LICENSE-APACHE travis/travis != root/root
E: fd: wrong-file-owner-uid-or-gid usr/share/doc/fd/LICENSE-MIT 2000/2000
E: fd: bad-owner-for-doc-file usr/share/doc/fd/LICENSE-MIT travis/travis != root/root
E: fd: wrong-file-owner-uid-or-gid usr/share/doc/fd/README.md 2000/2000
E: fd: bad-owner-for-doc-file usr/share/doc/fd/README.md travis/travis != root/root
E: fd: wrong-file-owner-uid-or-gid usr/share/fish/ 2000/2000
E: fd: wrong-file-owner-uid-or-gid usr/share/fish/completions/ 2000/2000
E: fd: wrong-file-owner-uid-or-gid usr/share/fish/completions/fd.fish 2000/2000
E: fd: wrong-file-owner-uid-or-gid usr/share/man/ 2000/2000
E: fd: wrong-file-owner-uid-or-gid usr/share/man/man1/ 2000/2000
E: fd: wrong-file-owner-uid-or-gid usr/share/man/man1/fd.1 2000/2000
E: fd: wrong-file-owner-uid-or-gid usr/share/zsh/ 2000/2000
E: fd: wrong-file-owner-uid-or-gid usr/share/zsh/vendor-completions/ 2000/2000
E: fd: wrong-file-owner-uid-or-gid usr/share/zsh/vendor-completions/_fd 2000/2000
E: fd: manpage-not-compressed usr/share/man/man1/fd.1
```

Unfortunately, the file owner problems lead to a bug where `zsh` will not parse the autocompletion file and print a warning:
```
zsh compinit: insecure files, run compaudit for list.
Ignore insecure files and continue [y] or abort compinit [n]? 
```

I'm already working on a fix.

**Comment by sharkdp** [comment]
The bug has been fixed for future releases.

I have also update the `fd_6.2.0_amd64.deb` package for the current release.

**Comment by kbenzie** [comment]
This also affects [fd-musl_6.2.0_amd64](https://github.com/sharkdp/fd/releases/download/v6.2.0/fd-musl_6.2.0_amd64.deb), I think it also needs updated. I've not checked the others but it may also affect them.

**Comment by sharkdp** [comment]
Yes, it will affect all deb packages (except the one I updated) - sorry for that. This should be fixed with the next release (and is easy to fix via `sudo chown -R root:root ...`)

**Comment by sharkdp** [comment]
Fix released in v6.3.0
